deploy/charts/external-secrets/templates/deployment.yaml

```diff
@@ -87,7 +87,7 @@ spec:
           {{- end }}
           {{- end }}
           ports:
-            - containerPort: {{ .Values.prometheus.service.port }}
+            - containerPort: 8080
```

Comment:
When we hardcode the metrics port to `8080` (in the other deployments as well) then in `cert-controller-service.yaml` the `Values.certController.metrics.service.port` can be potentially misconfigured.

From a user perspective i would say we have to have a way to configure the metrics listen port per deployment (some ppl :grimacing: run it in `hostNetwork=true` and need to avoid port conflicts).

I would make the the metrics listen port configurable for all deployments:

```yaml
webhook:
  # ...
  metrics:
    # this should be added.
    # should be an object so we can at some point add a `address` field if needed
    listen:
      port: 8080
    service: 
      enabled: false
      port: 8080 # the service port, not the container port
```

Reply:
hey @moolen , I left this issue as-is because I saw you assigned it to yourself. Let me know if you want to fix the hardcoded port or you want me to work on it.

Reply:
please fix it :pray: :bow: 
I assigned it to myself in order to review it :)

Reply:
should be resolved now

Reply:
Hey @sebagomez i got some comments below :point_down: did you see them? :)